bands/models.py

```diff
@@ -18,81 +18,111 @@ class Event:
 class Band(MagModel):
     group_id = Column(UUID, ForeignKey('group.id'))
     event_id = Column(UUID, ForeignKey('event.id'), nullable=True)
+    info = relationship('BandInfo', backref='band', uselist=False)
```

Comment:
@EliAndrewC is this right? I looked at SQLAlchemy's docs, we want uselist=False for one-to-one relationships, yeah?